docs/metrics/ga.md

```diff
@@ -107,6 +107,24 @@ Here are the current events on the website as of this writing, grouped by their
 | Retire from Test Pilot | button click | Retire |
 | Shows upgrade notice | Upgrade Warning | upgrade notice shown |
 
+#### `Featured Experiment`
+
+| Description                      | `eventAction`         | `eventLabel`                  | Custom Dimensions      |
+| -------------------------------- | --------------------- | ----------------------------- | ---------------------- |
+| Click Featured Experiment Video  | video click           | Play Featured video           | [1, 2, 3, 12, 10, 11]  |
+| Click View Details               | link click            | View Featured details         | [1, 2, 3, 4, 10, 11]   |
+| Featured Experiment Button       | button click          | Manage Featured Button State  | [1, 2, 3, 4, 5, 10, 11]|
+| Click Cancel on tour dialogue    | button click          | cancel tour                   | [1, 2, 3, 10, 11]      |
+| Complete the tour                | button click          | complete tour                 | [1, 2, 3, 10, 11]      |
+| Click Take Tour on tour dialogue | button click          | take tour                     | [1, 2, 3, 10, 11]      |
+| Click next during Tour           | button click          | forward to step `n`           | [1, 2, 3, 10, 11]      |
+| Click back during Tour           | button click          | back to step `n`              | [1, 2, 3, 10, 11]      |
+| Click pagination dot during Tour | button click          | dot to step `n`               | [1, 2, 3, 10, 11]      |
+| Click General Terms link         | link click            | Open general terms            | [1, 2, 3, 10, 11]      |
+| Click General Privacy link       | link click            | Open general privacy          | [1, 2, 3, 10, 11]      |
+| Click Featured Privacy link      | link click            | Popup Featured privacy        | [1, 2, 3, 10, 11]      |
```

Comment:
I'd still like to see a way to segment the events here that take place on both featured and non-featured experiments.